.vscode/settings.json

```diff
@@ -1,6 +1,19 @@
 {
   "editor.defaultFormatter": "rvest.vs-code-prettier-eslint",
+  "editor.formatOnSave": false,
```

Comment:
This should be opt out per person, not as a org wide setting. I quite like formatOnSave on.

Reply:
```suggestion
```